### DISCUSSION TITLE
There is a big difference between the Chinese docs and the English docs

### DISCUSSION THREAD

**Discussion by hooray** [vuejs/router#2301] [Ideas]
![image](https://github.com/vuejs/router/assets/1574903/af9bba77-3831-4257-af84-2c641cd44383)

At the API part of the docs, some APIs are missing from the Chinese docs, and there are also some APIs that don't match the English docs descriptions.

I would like to ask if assistance is needed. I can participate in supplementing and proofreading Chinese docs

**Comment by posva** [comment]
Help in translating is always welcome. The English API docs are fully generated based on source code while the Chinese (and others) need to be manually updated
